vscode/package.json

```diff
@@ -260,26 +261,33 @@
         "watch": "webpack --mode development --watch",
         "test-watch": "tsc -p ./ -watch",
         "test-compile": "tsc -p ./",
-        "jenkins-test": "tsc -p ./ && node ./out/test/runTest.js",
+        "jenkins-test": "tsc -p ./ && ts-node ./test/runTest.ts",
         "lint": "eslint src --ext ts",
         "lint-fix": "eslint src --ext ts --fix",
         "package": "vsce package --yarn",
         "clean": "rm -rf ./out ./dist ./node_modules"
     },
     "devDependencies": {
+        "@babel/core": "^7.14.6",
+        "@babel/plugin-transform-modules-commonjs": "^7.14.5",
+        "@babel/preset-env": "^7.14.7",
+        "@babel/preset-typescript": "^7.14.5",
```

Comment:
I don't think we use babel anywhere, try and remove the `babel.config.js` config file and these imports.